src/server/utilities/mvtHeader/enabledExperimentsList.js

```diff
@@ -7,6 +7,12 @@ Add enabled experiments objects inside this array in this format:
   pageTypes: [],
 }
 */
-const enabledExperimentList = [];
+const enabledExperimentList = [
+  {
+    name: 'test_2',
+    services: ['pidgin'],
+    pageTypes: ['STY', 'article'],
```

Comment:
`STY` could probably be removed from here as only `article` and `topics` pages support MVT headers currently.